### PR TITLE
Test fixes and refactoring

### PR DESCRIPTION
**Pull request by spriadka**

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/connections/BridgesTestCase.java [hunk 1]
```diff
@@ -97,170 +99,234 @@ public static void afterClass() throws IOException, OperationException, CommandF
...
     @Category(Elytron.class)
     @Test
     public void testIllegalCombinationsForCredentialReference() throws Exception {
-        page.switchToCredentialReference();
-        new ElytronIntegrationChecker.Builder(client)
-                .address(BRIDGE_ADDRESS)
-                .configFragment(page.getConfigFragment())
-                .build()
-                .testIllegalCombinationCredentialReferenceAttributes();
+        final ModelNodeResult bridgePassword = operations.readAttribute(BRIDGE_ADDRESS, PASSWORD);
```
Comment (rhatlapa) [inline basic/src/test/java/org/jboss/hal/testsuite/test/configuration/messaging/connections/BridgesTestCase.java]: You should ignore default values => ReadAttributeOption.NOT_INCLUDE_DEFAULTS

File: common/src/main/java/org/jboss/hal/testsuite/fragment/config/messaging/MessagingPropertyWizard.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package org.jboss.hal.testsuite.fragment.config.messaging;
+
+import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+
+public class MessagingPropertyWizard extends WizardWindow {
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/fragment/config/messaging/MessagingPropertyWizard.java]: javadoc please

**Review by rhatlapa — CHANGES_REQUESTED** [review]
see my comments.

File: common/src/main/java/org/jboss/hal/testsuite/fragment/config/messaging/MessagingPropertyWizard.java (hunk 2 above)
Comment (spriadka) [reply common/src/main/java/org/jboss/hal/testsuite/fragment/config/messaging/MessagingPropertyWizard.java]: @rhatlapa see fa95e1ad2fb0a15fe6990dc636be59b1357ad68a

**Comment by spriadka** [comment]
@rhatlapa can you check again with fa95e1a?

File: common/src/main/java/org/jboss/hal/testsuite/fragment/config/messaging/MessagingPropertyWizard.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package org.jboss.hal.testsuite.fragment.config.messaging;
+
+import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+
+/**
+ * Class representing dialog displayed when user wants to add new property in messaging configuration subsystem
+ */
+public class MessagingPropertyWizard extends WizardWindow {
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/fragment/config/messaging/MessagingPropertyWizard.java]: This looks like duplicate of `org.jboss.hal.testsuite.fragment.config.resourceadapters.ConfigPropertyWizard` shouldn't these two be unified in one?

**Comment by rhatlapa** [comment]
Only one item, otherwise LGTM

**Comment by spriadka** [comment]
@rhatlapa Minor changes were applied due to the ConfigPropertyWizard fixes. Shall I merge after squash?

File: common/src/main/java/org/jboss/hal/testsuite/fragment/config/resourceadapters/ResourceAdapterPropertyWizard.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
+package org.jboss.hal.testsuite.fragment.config.resourceadapters;
+
+import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+
+/**
+ * Class representing a dialog window for creating new resource adapter property
+ */
+public class ResourceAdapterPropertyWizard  extends WizardWindow {
+    private static final String NAME = "name";
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/fragment/config/resourceadapters/ResourceAdapterPropertyWizard.java:9]: question now is which one is more generally used, the one with "name" or the one with "key"?

**Comment by rhatlapa** [comment]
LGTM, if you are confident that the change in `ConfigPropertyWizard` will not break anything, then squash and I shall merge.

File: common/src/main/java/org/jboss/hal/testsuite/fragment/config/resourceadapters/ResourceAdapterPropertyWizard.java (hunk 4 above)
Comment (spriadka) [reply common/src/main/java/org/jboss/hal/testsuite/fragment/config/resourceadapters/ResourceAdapterPropertyWizard.java:9]: The one with the "key"

**Comment by spriadka** [comment]
@rhatlapa ran tests, change in ConfigPropertyWizard didl not break any on them

**Comment by spriadka** [comment]
@rhatlapa ready to merge

**Review by rhatlapa — APPROVED** [review]

**Pull request merged**